demos/speech_recognition_demo/python/speech_recognition_demo.py

```diff
@@ -27,26 +26,24 @@ def build_argparser():
                              "The sample will look for a suitable IE plugin for this device. (default is CPU)")
     parser.add_argument('-m', '--model', type=str, metavar="FILENAME", required=True,
                         help="Path to an .xml file with a trained model (required)")
+    parser.add_argument('-m_decoder', '--model_decoder', type=str, metavar="FILENAME",
+                        help="Optional. Path to an .xml QuartzNet decoder. Makes treat --model as QuartzNet encoder")
     parser.add_argument('-L', '--lm', type=str, metavar="FILENAME",
-                        help="path to language model file (optional)")
-    parser.add_argument('-p', '--profile', type=str, metavar="NAME", required=True,
+                        help="path to language model file (optional). Ignored if --model_decoder is set")
```

Comment:
You shouldn't ignore it; you should make sure it's _not_ specified if it's not appropriate for the current model.